lib/record_store/cli.rb

```diff
@@ -104,7 +104,7 @@ def apply
       end
 
       zones.each do |zone|
-        abort("Attempted to apply invalid zone: #{zone.name}") unless zone.valid?
+        abort("Attempted to apply invalid zone: #{zone.name}: #{zone.errors.full_messages.join(',')}") unless zone.valid?
```

Comment:
Perhaps this belongs in its own PR as it's not directly related to adding a new record type.